lib/pg_ha_migrations/unsafe_statements.rb

```diff
@@ -127,10 +210,6 @@ def unsafe_partman_update_config(table, **options)
 
     config = PgHaMigrations::PartmanConfig.find(_fully_qualified_table_name_for_partman(table))
 
-    if !options[:automatic_maintenance].nil?
-      options[:automatic_maintenance] = options[:automatic_maintenance] ? "on" : "off"
-    end
```

Comment:
I think this snuck in a previous PR: https://github.com/braintree/pg_ha_migrations/pull/117

I started to add this feature and realized a) this isn't something folks would be doing regularly and b) this was outside the scope of the PR. I guess I never fully cleaned up the changes, so doing so now.